platform/pulpcore/app/models/storage.py

```diff
@@ -1,400 +1,110 @@
 import os
 import errno
-import io
-import stat
-
-from datetime import datetime
-from uuid import uuid4
 
 from django.conf import settings
 from django.core.files import File
-from django.core.files.storage import Storage
-from django.utils.deconstruct import deconstructible
+from django.core.files.storage import FileSystemStorage
 
 
-class FileContent(File):
+class FileSystem(FileSystemStorage):
     """
-    An in-memory file.
-    Designed to be used by FileField when content is known.
-
-    Attributes:
-        content (str): The actual content.
-
-    Examples:
-        >>>
-        >>> from django.db import models
-        >>>
-        >>> class Person(models.Model):
-        >>>     ssl_certificate = models.FileField()
-        >>>
-        >>> person = Person()
-        >>> certificate = 'PEM ENCODED CERTIFICATE HERE'
-        >>> person.ssl_certificate = FileContent(certificate)
-        >>>
+    Modifies how Django saves files
```

Comment:
A better description of how this class differs from the Django one would be helpful.